gun-violence-dashboard/src/pages/Homepage.js

```diff
@@ -139,6 +187,7 @@ function Homepage() {
         <>
             <div className="m-3">
                 <h1 className="text-center">Number of deaths due to firearms per 100,000 population in the US</h1>
+                <h2 className="text-center">Click on state to learn more</h2>
```

Comment:
Styling of data could be improved here I think. Text alignment is off, and the stats could pop more.